setup.py

```diff
@@ -0,0 +1,111 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2020 CERN.
+#
+# invenio-exporter is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""Invenio module that adds Exporter programmatic API."""
+
+import os
+
+from setuptools import find_packages, setup
+
+readme = open('README.rst').read()
+history = open('CHANGES.rst').read()
+
+tests_require = [
+    'check-manifest>=0.25',
+    # coverage pinned because of https://github.com/nedbat/coveragepy/issues/716
+    'coverage>=4.0,<5.0.0',
+    'isort>=4.3.3',
+    'pydocstyle>=2.0.0',
+    'pytest-cov>=2.5.1',
+    'pytest-pep8>=1.0.6',
+    'pytest-invenio>=1.3.2',
+]
+
+extras_require = {
+    'docs': [
+        'Sphinx>=1.5.1',
```

Comment:
comment/heads up: if you see that errors fail with `outside of application context` or similar it might be this. For now I would say leave it as is, but in many (if not all) modules we have had to pin with `<3` version...